### PR TITLE
Broken down integration tests to smaller pieces

### PR DESCRIPTION
**Pull request by mihaib79**

### CODE REVIEW

File: integrationTests/multiShard/endOfEpoch/EpochStartChangeWithContinuousTransactionsInMultiShardedEnvironment/EpochStartChangeWithContinuousTransactionsInMultiShardedEnvironment_test.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
+package EpochStartChangeWithContinuousTransactionsInMultiShardedEnvironment
```
Comment (bogdan-rosianu) [inline integrationTests/multiShard/endOfEpoch/EpochStartChangeWithContinuousTransactionsInMultiShardedEnvironment/EpochStartChangeWithContinuousTransactionsInMultiShardedEnvironment_test.go]: same here. file name, package name and goimports

File: integrationTests/multiShard/endOfEpoch/EpochStartChangeWithoutTransactionInMultiShardedEnvironment/EpochStartChangeWithoutTransactionInMultiShardedEnvironment_test.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
+package EpochStartChangeWithoutTransactionInMultiShardedEnvironment
```
Comment (bogdan-rosianu) [inline integrationTests/multiShard/endOfEpoch/EpochStartChangeWithoutTransactionInMultiShardedEnvironment/EpochStartChangeWithoutTransactionInMultiShardedEnvironment_test.go]: same here

File: integrationTests/multiShard/endOfEpoch/ExecuteBlocksWithTransactionsAndCheckRewards/ExecuteBlocksWithTransactionsAndCheckRewards_test.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
+package ExecuteBlocksWithTransactionsAndCheckRewards
```
Comment (bogdan-rosianu) [inline integrationTests/multiShard/endOfEpoch/ExecuteBlocksWithTransactionsAndCheckRewards/ExecuteBlocksWithTransactionsAndCheckRewards_test.go]: same here

File: integrationTests/multiShard/endOfEpoch/ExecuteBlocksWithTransactionsAndCheckRewards/ExecuteBlocksWithTransactionsAndCheckRewards_test.go [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+import (
+	"context"
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/multiShard/endOfEpoch"
+	"testing"
+	"time"
+)
+
+func TestExecuteBlocksWithTransactionsAndCheckRewards(t *testing.T) {
```
Comment (bogdan-rosianu) [inline integrationTests/multiShard/endOfEpoch/ExecuteBlocksWithTransactionsAndCheckRewards/ExecuteBlocksWithTransactionsAndCheckRewards_test.go]: can you rename this test ? maybe add a `AtEndOfEpoch` suffix as there already is a test with the same name in a different package.
Also. I don't see any `require` or `assert` used in this test. I know it's not related to this PR, but can you check ?

File: integrationTests/multiShard/endOfEpoch/common.go [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+		integrationTests.SyncBlock(t, nodes, idxProposers, currentRound)
+		currentRound = integrationTests.IncrementAndPrintRound(currentRound)
+		currentNonce++
+	}
+	time.Sleep(time.Second)
+
+	return currentRound, currentNonce
+}
+
+func VerifyIfNodesHasCorrectEpoch(
```
Comment (bogdan-rosianu) [inline integrationTests/multiShard/endOfEpoch/common.go]: maybe rename to `VerifyThatNodesHaveCorrectEpoch`

File: integrationTests/p2p/antiflood/antiflooding/antiflooding_test.go [hunk 6]
```diff
@@ -1,7 +1,8 @@
-package antiflood
+package antiflooding
 
 import (
 	"fmt"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/p2p/antiflood"
```
Comment (bogdan-rosianu) [inline integrationTests/p2p/antiflood/antiflooding/antiflooding_test.go]: goimports

File: integrationTests/p2p/antiflood/blacklist/blacklist_test.go [hunk 7]
```diff
@@ -1,7 +1,8 @@
-package antiflood
+package blacklist
 
 import (
 	"fmt"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/p2p/antiflood"
```
Comment (bogdan-rosianu) [inline integrationTests/p2p/antiflood/blacklist/blacklist_test.go]: goimports

File: integrationTests/p2p/antiflood/common.go [hunk 8]
```diff
@@ -0,0 +1,65 @@
+package antiflood
+
+import (
+	"fmt"
```
Comment (bogdan-rosianu) [inline integrationTests/p2p/antiflood/common.go]: goimports

File: integrationTests/p2p/antiflood/common.go [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/ElrondNetwork/elrond-go/process/throttle/antiflood/floodPreventers"
+	"github.com/ElrondNetwork/elrond-go/storage/storageUnit"
+	"sync/atomic"
+	"time"
+)
+
+var DurationBootstrapingTime = 2 * time.Second
```
Comment (bogdan-rosianu) [inline integrationTests/p2p/antiflood/common.go]: why not const ?

File: integrationTests/resolvers/headers/headers_test.go [hunk 10]
```diff
@@ -1,7 +1,8 @@
-package resolvers
+package headers
 
 import (
 	"bytes"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/resolvers"
```
Comment (bogdan-rosianu) [inline integrationTests/resolvers/headers/headers_test.go]: goimports

File: integrationTests/resolvers/metablock/metablock_test.go [hunk 11]
```diff
@@ -1,7 +1,8 @@
-package resolvers
+package metablock
 
 import (
 	"bytes"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/resolvers"
```
Comment (bogdan-rosianu) [inline integrationTests/resolvers/metablock/metablock_test.go]: goimports

File: integrationTests/resolvers/miniblocks/miniblocks_test.go [hunk 12]
```diff
@@ -1,7 +1,8 @@
-package resolvers
+package miniblocks
 
 import (
 	"bytes"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/resolvers"
```
Comment (bogdan-rosianu) [inline integrationTests/resolvers/miniblocks/miniblocks_test.go]: goimports

File: integrationTests/resolvers/rewards/rewards_test.go [hunk 13]
```diff
@@ -1,7 +1,8 @@
-package resolvers
+package rewards
 
 import (
 	"bytes"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/resolvers"
```
Comment (bogdan-rosianu) [inline integrationTests/resolvers/rewards/rewards_test.go]: goimports

File: integrationTests/singleShard/block/common.go [hunk 14]
```diff
@@ -0,0 +1,6 @@
+package block
+
+import "time"
+
+var StepDelay = time.Second
```
Comment (bogdan-rosianu) [inline integrationTests/singleShard/block/common.go]: why not const ?

File: integrationTests/vm/arwen/badcontracts/badcontracts_test.go [hunk 15]
```diff
@@ -0,0 +1,43 @@
+package badcontracts
+
+import (
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm/arwen"
```
Comment (bogdan-rosianu) [inline integrationTests/vm/arwen/badcontracts/badcontracts_test.go]: goimports

File: integrationTests/vm/arwen/common.go [hunk 16]
```diff
@@ -0,0 +1,10 @@
+package arwen
+
+import (
+	"io/ioutil"
+	"path"
+)
+
+func GetBytecode(relativePath string) ([]byte, error) {
```
Comment (bogdan-rosianu) [inline integrationTests/vm/arwen/common.go]: maybe add mock comment to skip code style warnings. Something like `// GetBytecode --`

File: integrationTests/vm/arwen/erc20/erc20_test.go [hunk 17]
```diff
@@ -0,0 +1,107 @@
+package erc20
+
+import (
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm/arwen"
```
Comment (bogdan-rosianu) [inline integrationTests/vm/arwen/erc20/erc20_test.go]: goimports

File: integrationTests/vm/arwen/wasmer/wasmer_test.go [hunk 18]
```diff
@@ -1,8 +1,9 @@
-package arwen
+package wasmer
 
 import (
 	"encoding/hex"
 	"fmt"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm/arwen"
```
Comment (bogdan-rosianu) [inline integrationTests/vm/arwen/wasmer/wasmer_test.go]: goimports

File: integrationTests/vm/iele/common.go [hunk 19]
```diff
@@ -0,0 +1,41 @@
+package iele
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm"
+	"github.com/ElrondNetwork/elrond-go/process"
```
Comment (bogdan-rosianu) [inline integrationTests/vm/iele/common.go]: goimports

File: integrationTests/vm/iele/vmAgar/vmAgar_test.go [hunk 20]
```diff
@@ -1,9 +1,10 @@
-package mockVM
+package vmAgar
 
 import (
 	"crypto/rand"
 	"encoding/hex"
 	"fmt"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm/iele"
```
Comment (bogdan-rosianu) [inline integrationTests/vm/iele/vmAgar/vmAgar_test.go]: goimports

File: integrationTests/vm/iele/vmRunContract/vmRunContract_test.go [hunk 21]
```diff
@@ -1,14 +1,13 @@
-package mockVM
+package vmRunContract
 
 import (
 	"encoding/hex"
 	"fmt"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm/iele"
```
Comment (bogdan-rosianu) [inline integrationTests/vm/iele/vmRunContract/vmRunContract_test.go]: goimports

File: integrationTests/vm/arwen/arwenVM/arwenVM_test.go [hunk 22]
```diff
@@ -1,12 +1,11 @@
-package arwen
+package arwenVM
 
 import (
 	"crypto/rand"
 	"encoding/hex"
 	"fmt"
-	"io/ioutil"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm/arwen"
```
Comment (SebastianMarian) [inline integrationTests/vm/arwen/arwenVM/arwenVM_test.go]: goimports

File: integrationTests/p2p/antiflood/antiflooding/antiflooding_test.go [hunk 23]
```diff
@@ -2,12 +2,13 @@ package antiflooding
...
 import (
 	"fmt"
-	"github.com/ElrondNetwork/elrond-go/integrationTests/p2p/antiflood"
 	"math"
 	"sync/atomic"
 	"testing"
 	"time"
 
+	"github.com/ElrondNetwork/elrond-go/integrationTests/p2p/antiflood"
+
```
Comment (SebastianMarian) [inline integrationTests/p2p/antiflood/antiflooding/antiflooding_test.go]: remove empty line and run again goimports

File: integrationTests/p2p/antiflood/blacklist/blacklist_test.go [hunk 24]
```diff
@@ -2,12 +2,13 @@ package blacklist
...
 import (
 	"fmt"
-	"github.com/ElrondNetwork/elrond-go/integrationTests/p2p/antiflood"
 	"math"
 	"sync/atomic"
 	"testing"
 	"time"
 
+	"github.com/ElrondNetwork/elrond-go/integrationTests/p2p/antiflood"
+
```
Comment (SebastianMarian) [inline integrationTests/p2p/antiflood/blacklist/blacklist_test.go]: remove empty line and run again goimports

File: integrationTests/multiShard/endOfEpoch/common.go (hunk 5 above)
Comment (bogdan-rosianu) [reply integrationTests/multiShard/endOfEpoch/common.go]: not done yet

File: integrationTests/multiShard/endOfEpoch/ExecuteBlocksWithTransactionsAndCheckRewards/ExecuteBlocksWithTransactionsAndCheckRewards_test.go (hunk 4 above)
Comment (bogdan-rosianu) [reply integrationTests/multiShard/endOfEpoch/ExecuteBlocksWithTransactionsAndCheckRewards/ExecuteBlocksWithTransactionsAndCheckRewards_test.go]: not done yet

File: integrationTests/p2p/antiflood/blacklist/blacklist_test.go (hunk 24 above)
Comment (mihaib79) [reply integrationTests/p2p/antiflood/blacklist/blacklist_test.go]: done

**Pull request merged**